src/schema/validators/common.ts

```diff
@@ -512,6 +512,12 @@ export const MediaProps = {
         type: 'string',
     },
     dataLens: DataLensProps,
+    fullScreen: {
```

Comment:
there was no that field in validators at all?

Reply:
yes